matchbox_socket/src/webrtc_socket/native.rs

```diff
@@ -82,125 +83,231 @@ impl Signaller for NativeSignaller {
 
 pub(crate) struct NativeMessenger;
 
+impl PeerDataSender for UnboundedSender<Packet> {
+    fn send(&mut self, packet: Packet) -> Result<(), super::error::MessagingError> {
+        self.unbounded_send(packet).map_err(MessagingError::from)
```

Comment:
when does this happen? Can we drop the receiver while still having the sender available?

If so, maybe the outward error should either be PeerDisconnected, or SocketFutureDropped?

Reply:
Yeah, would only be if the receiver is closed, which happens when the peer_loop completes. Had it as this for consistency with Wasm, where (I think) there are more possible failure modes